### DISCUSSION TITLE
Any plans to switch from exoplayer to androidx.media3.exoplayer for Media Element?

### DISCUSSION THREAD

**Discussion by ne0rrmatrix** [CommunityToolkit/Maui#1400] [General]
Looks like google is no longer developing Exoplayer and is marking it as deprecated. The team at baseflow has a PR out discussing the switch to androidx.media3.exoplayer and it is about 55 percent done. Will any one be investigating the possibility of switching to new nuget? This will most likely not be for a while as it is still being worked on but I thought I would give everyone a heads up in case no one knew what was planned with exoplayer.

[https://github.com/Baseflow/ExoPlayerXamarin/issues/157#issue-1635594760](https://github.com/Baseflow/ExoPlayerXamarin/issues/157#issue-1635594760)

**Comment by TheCodeTraveler** [comment]
Hey @ne0rrmatrix! I've just opened a Proposal for us to port ExoPlayer to Media3: https://github.com/CommunityToolkit/Maui/issues/1511


Short answer - yes, we will be porting ExoPlayer Media3.

However, we are currently blocked because there is no Media3 Binding library. I've included more details in the Proposal including links to possible future Media3 binding libraries if you'd like to take a look!

**Reply by ne0rrmatrix** [reply]
TY for the Proposal and I will keep watching the ExoplayerXamarin and see what they end up choosing to do.
